docs/internals/contributing/bugs-and-features.txt

```diff
@@ -142,9 +142,9 @@ Although these votes on |django-developers| are informal, they'll be taken very
 seriously. After a suitable voting period, if an obvious consensus arises we'll
 follow the votes.
 
-However, consensus is not always possible. If consensus cannot be reached, or
+However, the consensus is not always possible. If consensus cannot be reached, or
 if the discussion toward a consensus fizzles out without a concrete decision,
-the decision may be deferred to the :ref:`technical board <technical-board>`.
+the decision may be referred to the :ref:`technical board <technical-board>`.
```

Comment:
That's not a typo.

Reply:
Reverting... :D

Reply:
I agree with Mariusz this is unnecessary to edit.